### DISCUSSION TITLE
Not able to see the boss-bridge-audit solutions

### DISCUSSION THREAD

**Discussion by akhilmanga** [Cyfrin/security-and-auditing-full-course-s23#134] [Q&A] [answered]
<img width="1105" alt="Screenshot 2024-02-26 at 4 35 55 PM" src="https://github.com/Cyfrin/security-and-auditing-full-course-s23/assets/116151859/7c524b40-3683-4e40-a6e3-ca6b7d7ff938">

I took gap because of my college exams. I'm back now @cromewar

**Comment by EngrPips** [comment, marked as answer]
<img width="1204" alt="Screenshot 2024-02-26 at 12 19 46 PM" src="https://github.com/Cyfrin/security-and-auditing-full-course-s23/assets/95760918/2ab9baf0-398d-465d-9127-798c1c5d7e8b">

Hello friend @akhilmanga you should change the branch to the audit-data branch as shown above and you should be able to find solutions
